src/andromede/input_converter/src/cc_configuration/batteries.yaml

```diff
@@ -0,0 +1,119 @@
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+# Configuration file to convert a complex modelling artefact in an Antares study into an Andromede model.
+
+loop: 
+    
+  iteration_key: zone #Key on which we iterate (for a given list of zone, we want to execute a loop : for zone in list_zone)
+  #In the following, the notation $zone$ means the value of the string variable "zone"
+  andromede-model: antares-historic.short-term-storage #we want to create components of such a model, with id being indexed by the iteration_key
+  andromede-component-id : battery_$zone$
+
+      
+  andromede-model-parameters-to-set-a-priori:
+    #TODO : we still have to see if the following parameters can be set a priori or have to be read in the Antares study,during the conversion.
+    #- id: lower_rule_curve
+    #- id: lower_rule_curve
+    #- id: p_max_injection_modulation  # Read in p_max_injection 
+    #- id: p_max_withdrawal_modulation # Read in p_max_withdrawal
+    
+    
+    - id: initial_level
+      time-dependent: false
+      scenario-dependent: false
+      value: -1.0 #TODO : The initial stock level constraint must be disabled or deleted in the Andromede ST storage model
```

Comment:
Do you think it is better to write a specialized short term storage models for batteries or we can keep the current one ? And usually the `initial_level` is at 0.5 ? Otherwise it may be optimized, in this case, we definitely need a different model